test/ios-project-service.ts

```diff
@@ -0,0 +1,148 @@
+/// <reference path=".d.ts" />
+"use strict";
+
+import Future = require("fibers/future");
+import * as path from "path";
+import temp = require("temp");
```

Comment:
Other tests can be rewritten in the future. Do not depend on side effects of unrelated code.